rllib/models/modelv2.py

```diff
@@ -223,6 +232,26 @@ def from_batch(self, train_batch, is_training=True):
             i += 1
         return self.__call__(input_dict, states, train_batch.get("seq_lens"))
 
+    def get_view_requirements(self, is_training=False):
```

Comment:
This may get called each inference and training call (or just once, we'll see) to return the required view specs.